src/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -1,54 +1,108 @@
-import React from 'react'
+import React, {useEffect, useState} from 'react'
 import {defineMessages, useIntl} from 'react-intl'
-import {FlatList, StyleSheet, View} from 'react-native'
+import {StyleSheet, TouchableOpacity, View, ViewProps} from 'react-native'
 import Markdown from 'react-native-easy-markdown'
 
 import {useLanguage} from '../../i18n'
-import {LanguageListItem} from './LanguageListItem'
+import {Button} from '../Button'
+import {Icon} from '../Icon'
+import {LanguagePickerList} from './LanguagePickerList'
 
 const INCLUDED_LANGUAGE_CODES = ['en-US', 'ja-JP']
 
-export const LanguagePicker = () => {
-  const languageContext = useLanguage()
+export const Row = ({style, ...props}: ViewProps) => <View {...props} style={[styles.row, style]} />
 
+export const LanguagePicker = ({onPressConfirmButtonCallback, buttonLabel, noWarningMessage = false}: Props) => {
+  const languageContext = useLanguage()
```

Comment:
this breaks encapsulation
pretend you don't know what's on the inside of any given function